app/crud.py

```diff
@@ -70,6 +73,7 @@ async def create_answer_table_entry(db: Database, log_entry: schemas.LogItem):
     return res
 
 
+# :index:`TODO`: **I think the idea here**, but the implementation will still need some special cases for getting the specific data for all the question types.
```

Comment:
My thinking here is that we always return the entire row for any given question so that it **does** have any extra columns added for a particular question type.

Reply:
That seems reasonable. I think most of this will be resolved with good tests.